website/source/api/snapshot.html.md

```diff
@@ -60,7 +60,7 @@ The table below shows this endpoint's support for
 With a custom datacenter:
 
 ```text
-$ curl https://consul.rocks/v1/snapshot?dc=my-datacenter
+$ curl https://consul.rocks/v1/snapshot?dc=my-datacenter -o snapshot.tgz
 ```
 
 ### Sample Response
```

Comment:
We can remove this Sample Response section now and instead add a line stating that the snapshot is saved to `snapshot.tgz`

Reply:
I think that would be a good idea indeed! I thought about also showing the contents of the snapshot, but that seems like too much / off topic.

Reply:
Okay, added a note about the file output.